docs/configuration.rst

```diff
@@ -53,8 +55,18 @@ In the configuration file itself, there are multiple options to vary a parameter
 Override configuration
 ---------------------------
  
-If tools of one module, let's call this module **A** for now, are called from another module **B**, there is the option to include an *A_override* section in the 
+If tools of one module, let's call this module **A** for now, are called from another module **B**, there is the option to include an *collectionName_A_override* section in the
 configuration file of module **B**. In this case, the default configuration of module **A** is read and the parameters in the *A_override* section 
 in the module **B** configuration file are used to update the configuration settings of module **A**. This has the advantage of gathering all 
-the configuration parameters used for one task in one configuration file. Find below an example of: 
+the configuration parameters used for one task in one configuration file.
+An example of this usage can be found in ``ana1Tests/energyLineTestCfg.ini``.
+
+Additionally, there is the option to (re-)write ``local_moduleNameCfg.ini`` configuration files for all modules
+where a corresponding *collectionName_moduleName_override* section is provided by using
+the function: :py:func:`in3Utils.cfgHandling.rewriteLocalCfgs`. An example of this functionality is provided in
+:py:mod:`runScripts.runPlotContoursFromAsc.py` with the respective ``runPlotContoursFromAscCfg.ini`` file.
+
+.. Note::
+   By using :py:func:`in3Utils.cfgHandling.rewriteLocalCfgs`, already existing ``local_moduleNameCfg.ini`` files will be
+   overwritten for the respective modules.
```

Comment:
Since this is not how it works at the moment, we could / should take this out, right?